packages/experimental/Text/src/Text.tsx

```diff
@@ -50,6 +51,12 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     ? 'right'
     : align;
 
+  const onAccTap = onAccessibilityTap
+    ? onAccessibilityTap
+    : React.useCallback((event?) => {
+        props.onPress(event);
+      }, []);
```

Comment:
I think props.onPress needs to be provided in the brackets as a cache key for the callback to update properly